introduction.md

```diff
@@ -1 +1,5 @@
 Hi, my name is grossb1.  My favorite code school path is Ruby
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?